app/src/main/java/com/example/bookclub/LibraryActivity.java

```diff
@@ -1,14 +1,78 @@
 package com.example.bookclub;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
 
 import android.os.Bundle;
+import android.view.View;
+
+import com.example.bookclub.Adapters.LibraryItemAdapter;
+import com.example.bookclub.models.LibraryItem;
+import com.parse.FindCallback;
+import com.parse.GetCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class LibraryActivity extends AppCompatActivity {
+    private RecyclerView rvLibraryItem;
+    protected LibraryItemAdapter libraryItemAdapter;
+    protected List<LibraryItem> allLibraryItems;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_library);
+        setContentView(R.layout.library_item);
+
+
+        rvLibraryItem = findViewById(R.id.rvLibraryItem);
+        allLibraryItems = new ArrayList<>();
+        libraryItemAdapter = new LibraryItemAdapter(this, allLibraryItems);
+        rvLibraryItem.setAdapter(libraryItemAdapter);
+        rvLibraryItem.setLayoutManager(new LinearLayoutManager(this));
+        populateLibraryItem();
+    }
+
+    private void populateLibraryItem(){
+        ParseUser currentUser = ParseUser.getCurrentUser();
+        JSONArray itemsInLibrary = currentUser.getJSONArray("LibraryItem");
+        if(itemsInLibrary == null){
+            return;
+        }
+        for (int i = 0; i < itemsInLibrary.length(); i++){
```

Comment:
Space before '{'. You can just run the formatter and it will make the changes for you.